docs/r_sig/23_11_06_renv/index.qmd

```diff
@@ -5,7 +5,7 @@ author:
   - name: Nicklas Hafiz
   - affiliation: PhD student at the IQB, Methods team
 date: 10-23-2023
-categories: [R, renv, reproducability, R-SIG]
+categories: [R, reproducability, R-SIG]
```

Comment:
I think renv as a category may be more useful, because when i look for it, I would look for renv, not reproducibility (and it's spelled with an 'i')